sklearn/manifold/tests/test_mds.py

```diff
@@ -69,3 +75,31 @@ def test_normalize_metric_warning():
     sim = np.array([[0, 5, 3, 4], [5, 0, 2, 2], [3, 2, 0, 1], [4, 2, 1, 0]])
     with pytest.raises(ValueError, match=msg):
         mds.smacof(sim, metric=True, normalized_stress=True)
+
+
+@pytest.mark.parametrize("metric", [True, False])
+def test_normalized_stress_default_change(metric):
+    msg = "The default value of `normalized_stress` will change"
+    sim = np.array([[0, 5, 3, 4], [5, 0, 2, 2], [3, 2, 0, 1], [4, 2, 1, 0]])
+    est = mds.MDS(metric=metric)
+    with pytest.warns(FutureWarning, match=msg):
+        mds.smacof(sim, metric=metric)
+    with pytest.warns(FutureWarning, match=msg):
+        est.fit(sim)
+
+
+@pytest.mark.parametrize("metric", [True, False])
+def test_normalized_stress_auto(metric, monkeypatch):
+    rng = np.random.RandomState(0)
+    X = rng.randn(3, 3)
```

Comment:
With `(3, 3)`, a false warning is raised here:

https://github.com/scikit-learn/scikit-learn/blob/4ee3fdd85c8d78da24c166a4127e8bc7bc8dd469/sklearn/manifold/_mds.py#L584-L586

To work around that we can:

```suggestion
    X = rng.randn(4, 3)
```